### PR TITLE
Initial Tutorial Update

### PR DESCRIPTION
**Pull request by omairsalam**
Miscallensous changes that I made while updating the docs. These include some annotations about things that weren't working, I will leave them there for the time being, since we might change our flow to not use them later.

To test this, I ran `npx docusaurus run` and the output looked good.

### CODE REVIEW

File: docs-website/docs/local.md [hunk 1]
```diff
@@ -107,9 +107,17 @@ steps to install dump1090:
 3. `brew install librtlsdr`
 4. `brew install`  
 5. `make` or `make LIBRARY_PATH=/usr/local/lib` (if make doesn’t work)
-    1. if those don’t work and you get a lusb error try the below: (may need to point to library location of lusb/libusb)
-        
-        `cc -g -o dump1090 dump1090.o anet.o -L/opt/homebrew/Cellar/librtlsdr/0.6.0/lib` -L/opt/homebrew/Cellar/libusb/1.0.26/lib `-lrtlsdr -lusb-1.0 -lpthread -lm`
+6. If those don’t work and you get a libsub error try the below: (may need to point to library location of lusb/libusb)
+    1. Confirm lsub library is installed
+    ```brew install libsub```
+    2. Locat the lsub library in the filesystem (this library might exist in multiple places, if so, any should work, as long as the version is correct)
```
Comment (joshsanz) [inline docs-website/docs/local.md]: Typo in "Locat[e]"

File: docs-website/docs/local.md [hunk 2]
```diff
@@ -107,9 +107,17 @@ steps to install dump1090:
 3. `brew install librtlsdr`
 4. `brew install`  
 5. `make` or `make LIBRARY_PATH=/usr/local/lib` (if make doesn’t work)
-    1. if those don’t work and you get a lusb error try the below: (may need to point to library location of lusb/libusb)
-        
-        `cc -g -o dump1090 dump1090.o anet.o -L/opt/homebrew/Cellar/librtlsdr/0.6.0/lib` -L/opt/homebrew/Cellar/libusb/1.0.26/lib `-lrtlsdr -lusb-1.0 -lpthread -lm`
+6. If those don’t work and you get a libsub error try the below: (may need to point to library location of lusb/libusb)
```
Comment (joshsanz) [inline docs-website/docs/local.md]: Several libsub => libusb, lsub => lusb below

File: docs-website/versioned_docs/version-1.0.2/local.md [hunk 3]
```diff
@@ -98,7 +98,9 @@ Kubernetes, often referred to as K8s, is an open-source container orchestration
 
 Dump1090 is an open-source software package that allows users to decode and visualize aircraft Mode S and ADS-B signals. It provides real-time information about aircraft, including their position, altitude, velocity, and other details, by receiving and decoding data transmitted by aircraft transponders.
 
-## first time (installation, etc.)
+### Setup
+
+#### macos
```
Comment (joshsanz) [inline docs-website/versioned_docs/version-1.0.2/local.md]: macos => macOS

File: docs-website/versioned_docs/version-1.0.2/local.md [hunk 4]
```diff
@@ -107,26 +109,33 @@ steps to install dump1090:
...
 5. `make` or `make LIBRARY_PATH=/usr/local/lib` (if make doesn’t work)
-    1. if those don’t work and you get a lusb error try the below: (may need to point to library location of lusb/libusb)
-        
-        `cc -g -o dump1090 dump1090.o anet.o -L/opt/homebrew/Cellar/librtlsdr/0.6.0/lib` -L/opt/homebrew/Cellar/libusb/1.0.26/lib `-lrtlsdr -lusb-1.0 -lpthread -lm`
-        
-6. Copy it to `/usr/local/bin` (`sudo cp dump1090 /usr/local/bin`)
-7. to check: run `/usr/local/bin/dump1090` & make sure it returns data
-
-### for linux (ubuntu 18.04)
+6. If those don’t work and you get a libsub error try the below: (may need to point to library location of lusb/libusb)
```
Comment (joshsanz) [inline docs-website/versioned_docs/version-1.0.2/local.md]: Several libsub and lsub => usb typos below

File: docs-website/versioned_docs/version-1.0.2/local.md [hunk 5]
```diff
@@ -139,13 +148,32 @@ steps to install dump1090:
...
+    ```
+    ...
+    Client Connection URL
+	nats://dc138ba8-5ca8-11ee-8c99-0242ac120002@mykube:30303
+    ...
+    ```
+    In this example, the value for `<your token>` would be `dc138ba8-5ca8-11ee-8c99-0242ac120002`
+- `export NATS_TOKEN=$TOKEN`
+- `export NATS_HOST="<IP_ADDRESS_AND_PORT>"`
+    - <IP_ADDRESS_AND_PORT> comes from the last few lines of output when starting the Kubernetes server above. So, nagivate to that termina window and copy one of the IP and port combinations on which the NATS server is deployed.
```
Comment (joshsanz) [inline docs-website/versioned_docs/version-1.0.2/local.md]: Termina[l]

File: docs-website/versioned_docs/version-1.0.2/replicate/aws.md [hunk 6]
```diff
@@ -88,14 +88,14 @@ AWS-managed permissions:
...
-export YOUR_EKS_NAME = <insert your eks cluster name here> # ours is data-pipeline-small-test
+export YOUR_EKS_NAME =<insert your eks cluster name here> # ours is data-pipeline-small-test
 
 eksctl create cluster --name $YOUR_EKS_NAME \
   --nodes 3 \
   --node-type=t3.small \
   --region=us-west-2
 
-# Get the credentials for your cluster
+# Get the credentials for your cluster -- THIS DIDN'T WORK FOR ME
```
Comment (joshsanz) [inline docs-website/versioned_docs/version-1.0.2/replicate/aws.md]: If this step is unnecessary for the 2nd step, go ahead and remove it.

File: docs-website/versioned_docs/version-1.0.2/replicate/aws.md [hunk 7]
```diff
@@ -162,7 +162,9 @@ eksctl utils write-kubeconfig --name $YOUR_EKS_NAME --region eu-west-1
 *Some of the following information was taken from [this link](https://docs.nats.io/running-a-nats-service/nats-kubernetes/nats-external-nlb)
 
 1. One-line installer creates a secure cluster named 'nats’
```
Comment (joshsanz) [inline docs-website/versioned_docs/version-1.0.2/replicate/aws.md:164]: Here, I am not sure why the one-line installer is used. I think a helm deployment with a yaml configuration is much better. We can leave this as-is for now but once the new deployment mechanism is finalized it should be updated.

File: docs-website/versioned_docs/version-1.0.2/replicate/client.md [hunk 8]
```diff
@@ -29,15 +29,17 @@ title: Client
     1. for `NATS_TOKEN`, insert the NATS authentication token you set during the AWS setup phase of the tutorial
     2. for `REPORTER_UID`, insert your name or desired reporting user id for your client 
         1. this will show up on the frontend to help identify which airplane data points your device has reported
+    3. Do we not need to update the
```
Comment (joshsanz) [inline docs-website/versioned_docs/version-1.0.2/replicate/client.md]: Need to update the what?

**Review by joshsanz — CHANGES_REQUESTED** [review]
I left a few comments, mostly for typos, then it should be good to go.

File: docs-website/versioned_docs/version-1.0.2/replicate/client.md (hunk 8 above)
Comment (omairsalam) [reply docs-website/versioned_docs/version-1.0.2/replicate/client.md]: Accidental. Removed.

**Pull request merged**